entity-framework/core/providers/devart/index.md

```diff
@@ -48,4 +48,5 @@ See the [Devart dotConnect Entity Framework documentation](https://www.devart.co
 
 ## Supported Platforms
 
-* .NET Framework (4.5.1 onwards)
+* .NET Framework 2.0, 3.0, 3.5, 4.0, 4.5, 4.6, 4.7
```

Comment:
But EF Core itself doesn't support .NET Framework 2.0 thru 4.5, so how can this provider?